doc/src/tutorial/features.rst

```diff
@@ -0,0 +1,18 @@
+.. _features:
```

Comment:
This should be something like `_tutorial-features` so that it is a cross reference name that is specific to the tutorial. Otherwise if someone somehow crossreferences "features" somewhere it will link here, but that might not be where they were trying to reference to.